File: yunva17/프로그래머스/Lv2/JadenCase 문자열 만들기.js
function solution(s) {
  let answer = '';
  let slist = s.split(' ');

  slist = slist.map(
    (item) => item.slice(0, 1).toUpperCase() + item.slice(1).toLowerCase()
  );

  console.log(slist);
  answer = slist.join(' ');
  return answer;
}

File: overthename/Programmers/level2/JadenCase문자열만들기.java
class Solution {
	public String solution(String s) {
		String answer = "";
		boolean first = true;
		String[] arr = s.toLowerCase().split("");

		for(String str : arr){

			answer+= (first? str.toUpperCase() :str);
			first = str.equals(" ")?true:false;
		}



		return answer;
	}
}